taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/OutboxRestService.java

```diff
@@ -0,0 +1,28 @@
+package pro.taskana.adapter.camunda.outbox.rest;
+
+import pro.taskana.adapter.camunda.outbox.rest.dto.ReferencedTaskDTO;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Path("/outbox")
+public interface OutboxRestService {
+
+
+    @Path("/getCreateEvents")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<ReferencedTaskDTO> getCreateEvents();
+
+
+    //In Progress, will be changed to POST + ability to process list of ids
+    @Path("/delete/{id}")
```

Comment:
The URL should be /delete?ids=<string containing a comma separated list of ids>
e.g. /delete?ids=3,7,5,43